.github/workflows/cli.yaml

```diff
@@ -162,28 +104,8 @@ jobs:
       steps:
             - uses: actions/checkout@v3
```

Comment:
Do we need this if this is done at the composite action?

Reply:
Yes, we need checkout to get the contents of the repo and then refer to the composition action